app/policies/events/rubygem_event_policy.rb

```diff
@@ -0,0 +1,23 @@
+class Events::RubygemEventPolicy < ApplicationPolicy
+  class Scope < Scope
+    def resolve
+      scope.all
```

Comment:
Is this, and other, `Scope` classes required to be defined if we're not really scoping anything?

Reply:
Yeah, honestly I was leaving theme here since it's part of the basic structure, but we aren't using them. I figured place holders don't hurt, but if we all agree it's unnecessary, I can removed them.

Reply:
For now I'm defaulting all scopes to most restrictive. If someone implements them, then they will have to update to return something other than `scope.none`. I think this is the most explicit way to do it for now until we know if they're used.